### ISSUE TITLE
Foundational Prototype Buy Package

### ISSUE THREAD

**Issue by Araxiel** [Araxiel/Dead_Money-A_Vicky3_Fallout_Mod#65]
To have pop needs work, I need to to add it all in the `buy_package.txt`.
And there are 99 wealth levels and having to write out the entire `buy_package` will take forever and is an absolute pain the butt.

Have asked on the discord a guy that made a kind of script, waiting to hear back from him.

Alternatively, perhaps create some kind of excel sheet to copy-paste things?

Ugh, dreading this truly

**Comment by Araxiel** [comment]
Is still missing robots luxury good

**Comment by Araxiel** [comment]
Once the FPB is further along and had some less broken testing, I'll check back on this and properly close it.

**Comment by HiddeLekanne** [comment]
Hi @Araxiel, developer of a pop_needs (and buildings) tool here (https://github.com/Babylon-Development-Victoria-3/Victoria-3-dev-utilities). did it help? Can you provide feedback if you have any? I like to know how the tool is used, and how it is useful.

**Comment by Araxiel** [comment]
> Hi @Araxiel, developer of a pop_needs (and buildings) tool here (https://github.com/Babylon-Development-Victoria-3/Victoria-3-dev-utilities). did it help? Can you provide feedback if you have any? I like to know how the tool is used, and how it is useful.

Heyo @HiddeLekanne, I saw the link to your tool in the Vic3 modding discord's tools section. I thought about giving it a spin, but as much as buy_packages are a pain in the butt, sorting out how to run your tool seemed also like a pain, and before I knew it, I just kinda edited it all by hand while listening to a podcast.

But this initial edit is just to get the game even running. As the mod first published version get's more serious and more finished and somewhat more concrete and balanced numbers are gonna get implemented, I'll absolutely will try to have a look at your tool, cause it looks very useful.

Edit: I also added a little blurb with a link to the tool's repo in the goods modding page I wrote yesterday, at the bottom on how to add buy_packages
https://vic3.paradoxwikis.com/Goods_modding#Creating_a_new_need
